lnwallet/chancloser/chancloser.go

```diff
@@ -705,6 +705,15 @@ func (c *ChanCloser) ProcessCloseMsg(msg lnwire.Message, remote bool) (
 		chancloserLog.Infof("ChannelPoint(%v): entering fee "+
 			"negotiation", c.chanPoint)
 
+		if remote && c.isChannelClean() {
```

Comment:
IIUC, we should have already called this (in the peer) since we receive a shutdown message that prompts the chan closer to be created in the first place.